usr/share/rear/backup/DUPLICITY/default/980_unmount_duplicity_path.sh

```diff
@@ -5,5 +5,5 @@ if [  -n "$BACKUP_DUPLICITY_NETFS_URL" -o -n "$BACKUP_DUPLICITY_NETFS_UMOUNTCMD"
 		BACKUP_DUPLICITY_NETFS_URL="var://BACKUP_DUPLICITY_NETFS_UMOUNTCMD"
 	fi
 
-	umount_url $BACKUP_DUPLICITY_NETFS_URL $BUILD_DIR/outputfs
+	umount_url "$BACKUP_DUPLICITY_NETFS_URL" $BUILD_DIR/outputfs
```

Comment:
`$BUILD_DIR/outputfs` should be quoted as well for consistency with other changes